docker/README.md

```diff
@@ -1,35 +1,47 @@
 # Docker
```

Comment:
I was thinking that the individual readme files at docker/server and docker/ui could be consolidated into the readme under docker.